doc/governance/project-docs/GOVERNANCE.md

```diff
@@ -0,0 +1,54 @@
+# Governance Policy
+
+The "Project" is herein defined as the activities related to this specific GitHub repository [`HoloViz`](https://github.com/holoviz/holoviz), within the `HoloViz` GitHub Organization. This purpose of this repository is primarily to host the development activities of the HoloViz.org website. This repository also hosts the HoloViz Organization governance documents, that apply to the wider HoloViz ecosystem.
+
+This document provides the governance policy for the Project. Members agree to this policy and to abide by all Project polices, including the [code of conduct](../org-docs/CODE-OF-CONDUCT.md), [trademark policy](../org-docs/TRADEMARKS.md), and [antitrust policy](../org-docs/ANTITRUST.md) by adding their name to the [MEMBERS.md file](./MEMBERS.md).
+
+## 1. Project Member Roles.
+
+This project may include the following roles. Additional roles may be adopted and documented by the Project. All member role terms may be for life, as long as the members adhere to the Organization's policies.
+
+**1.1 Maintainer**. Maintainers are responsible for organizing activities around developing, maintaining, and updating the Project. Maintainers are also responsible for determining consensus. Maintainers are contributors that have an ongoing relationship to a project, and typically have GitHub merge permissions. Maintainers may be involved in different aspects of the Project, such as the code or the documentation. This Project may add or remove Maintainers with the approval (or simple majority vote if consensus is not reached) of the current Maintainers. When submitting substantial pull-requests, Maintainers should seek code review by another Maintainer. Maintainers may review and merge (or reject) pull-requests from Contributors, and seek Maintainers' consensus when appropriate. If a Maintainer is unresponsive at their listed handle for more than three months, the Maintainers may vote to remove the member.
+
+**1.2 Project Owner**. The Project Owner is a Maintainer that will have the final say on the project direction and management, unless overridden. By default and whenever possible, the Project Owner should defer to the consensus forming process of the Maintainers. Maintainers may override a Project Owner by 3/4 vote. Maintainers may elect or remove a Project Owner by 4/5 vote. During their term, the Project Owner should maintain a high level of involvement in most types of activity with the Project, such as advocacy, planning, support, review, and development. The Project Owner also assumes Lead Maintainer and Release Manager duties by default, if those roles are vacant. A Project Owner may resign at any time. If a Project Owner is unresponsive at their listed handle for more than three months, the Maintainers may vote to remove the member.
```

Comment:
I agree with reasonably high thresholds here but some of these thresholds may not actually be attainable, i.e. what happens for projects with < 5 maintainers, there could never be a vote to remove the project owner. Also while I obviously want to strive for consensus and consider such a situation highly unlikely at least in the state of the project I would probably not actually agree to be bound by a vote to overrule my decisions or be removed as owner from either Panel or Lumen.

Reply:
Thanks for the questions! This is the exact type of discussion I was hoping to get :)


re: inability to meet threshold... Indeed, I was thinking that if there are fewer than 5 people as maintainers on a project, then it should not be possible to remove a project owner as long as the project owner does not vote against themself. I was thinking that smaller projects should be handled differently than larger ones, with project owners exerting more unchecked control if there are only a few people involved in a project. But I've somewhat arbitrarily drawn that line at a number that I think would be reasonable. Any suggestions for an alternative?

re: preferring to be project BDFL... I suppose there may be flexibility within the proposed model for a BDFL at the project level with some additional clarifications. In other BDFL (or BDFL-like) projects, like dask, matplotlib, and pandas, the BDFL is actually at the org level, not the project level. So in their case, it's clear that the BDFL has full unchecked power over everything. If you want to go this route, we should think carefully and add to the CHARTER.md to clearly define the roles and powers of the steering committee vs a project's BDFL. Would you be comfortable with the existence of any threshold for an override or removal of project BDFL if it needed to pass both with maintainers of the project and also the Steering Committee? Alternatively, we could define specific vote topics to be only under the jurisdiction of the BDFL (like software related). Another alternative is to do something like what conda does with Federated projects, where essentially there are certain projects under the 'conda umbrella' but they are in (almost) full control of the project authors - essentially removing the authority of a steering committee for a project. If this type of thing (federated project) might be closer to what you are leaning towards, even then the Project would need to adhere to the code of conduct, and a serious violation of the code of conduct could possibly result in a permanent band, as currently written.

Reply:
Seems like the requirement for >4 core members to override a project owner could take care of the wish to have BFDL status, in that admitting enough official Maintainers will inherently allow the possibility of eventual removal, and so if that is not desired, don't accept that many Maintainers (but instead have some other category without such voting rights). Basically, either allow people to share ownership, or not. If they are allowed to truly share ownership, that has to include getting rid of the BDFL. If they are not, don't pretend people have ownership; they don't.

Reply:
@philippjfr, I hear you. As we've started doing in comments later on in this PR, I think clarifying the jurisdiction of the Steering Committee will make it easier to form the Project governance (of Lumen, Panel, etc) to your liking.

Reply:
To address the concern regarding the possibility for Maintainers to remove a Project Lead (now called Project Director) with 3/4 vote (rather than the intended higher threshold of 4/5) so long as they first add a new Maintainer who would take their side, I have revised the governance such that the Maintainers cannot override a Project Director’s veto on matters about admission of new members.